backend/chalice/api_server/Makefile

```diff
@@ -40,6 +40,7 @@ package: build-chalice-config
 	mkdir -p chalicelib/corpora
 	cp -R ../../corpora/common chalicelib/corpora
 	cp -R ../../corpora/lambdas chalicelib/corpora
+	cp -R ../../corpora/dataset_processing chalicelib/corpora
```

Comment:
do you still need to include data_processing?

Reply:
oh good catch-- removed